helpers.h

```diff
@@ -1,16 +1,17 @@
 // To be Included the header file "bmp.h"
+#include "bmp.h"
 
 // Convert Image to grayscale
-// write protopype below this comment
+ void grayscale(int, int, RGBTRIPLE **);
 
 // Reflect image horizontally
-// write protopype below this comment
+ void reflect(int, int, RGBTRIPLE **);
 
 // Detect edges
-// write protopype below this comment
+ void edges(int, int, RGBTRIPLE **);
 
 // Blur image
-// write protopype below this comment
+ void blur(int, int, RGBTRIPLE **);
 
 // Sepia image
-// write protopype below this comment
\ No newline at end of file
+ void sepia(int, int,RGBTRIPLE **);
```

Comment:
```suggestion
 void sepia(int, int, RGBTRIPLE **);
```
You didn't add space? Also please add a new line every time you reach the end of the file.